src/Pages/Cart/Components/Product.js

```diff
@@ -0,0 +1,119 @@
+import { useWish } from "../../Products/Context/WishContext";
+import { useProduct } from "../../Products/Context/ProductContext";
+import { useCart } from "../../Products/Context/CartContext";
+
+import { useAuthData } from "../../../Context/AuthContext";
+
+import { removeCart, removeWishlist } from "../../../ApiService";
+
+export const Product = ({
+  _id,
+  image,
+  title,
+  price,
+  mrpPrice,
+  isLiked,
+  qty,
+}) => {
+  const { WishlistDispatch } = useWish();
+  const { ProductDispatch } = useProduct();
+  const { CartDispatch } = useCart();
+  const { userAuth } = useAuthData();
+  const { isUserLoggedIn, encodedToken } = userAuth;
```

Comment:
can but into single line

Reply:
thanks this pointing out. you will see change in next commit